pkg/gridtypes/zos/types.go

```diff
@@ -12,10 +12,14 @@ const (
 	ZMountType gridtypes.WorkloadType = "zmount"
 	// NetworkType type
 	NetworkType gridtypes.WorkloadType = "network"
-	// ZDBType type
-	ZDBType gridtypes.WorkloadType = "zdb"
+	// NetworkLightType type
+	NetworkLightType gridtypes.WorkloadType = "network-light"
+	// ZDBLightType type
+	ZDBLightType gridtypes.WorkloadType = "zdb-light"
```

Comment:
Why do we need zdb-light? is it because it doesn't have ygg IP? 

I think we should only use the "-light" suffix if the data type for input or output changes (like vm and network) but I don't believe this applies to zdb